src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerRequestForInformation.java

```diff
@@ -40,14 +40,18 @@ public class CaseworkerRequestForInformation implements CCDConfig<CaseData, Stat
     public static final String CASEWORKER_REQUEST_FOR_INFORMATION = "caseworker-request-for-information";
     public static final String REQUEST_FOR_INFORMATION_NOTIFICATION_FAILED_ERROR
         = "Unable to send Request for Information Notification for Case Id: ";
-    public static final String NO_VALID_EMAIL_ERROR
-        = "You cannot send an email because no email address has been provided for ";
+    public static final String YOU_CANNOT_SEND_AN_EMAIL = "You cannot send an email because ";
+    public static final String NO_VALID_EMAIL_ERROR = YOU_CANNOT_SEND_AN_EMAIL + "no email address has been provided for ";
+    public static final String APPLICANT_IS_OFFLINE = " is offline";
+    public static final String SOLICITOR_IS_OFFLINE = " has not agreed to receive emails";
```

Comment:
I thinlk it'd be easier if we didn't add string constants in the actual code and just defined what's needed in the first place, there isn't that many combinations to warrant it

Reply:
I've refactored this down to 2x predefined errors + appropriate parties.  I think that's a good compromise.  Otherwise it's double the number of predefined strings and any changes to the wording need to be made up to 12 times instead of once.